src/utilities/data/DataEnums.hpp

```diff
@@ -71,9 +73,9 @@ OPENSTUDIO_ENUM(FuelType,
   ((FuelOil_2)(FuelOilNo2))
   ((Propane))
   ((Water))
-  ((Steam))
+  ((Steam)(DistrictHeatingSteam))
   ((DistrictCooling))
-  ((DistrictHeating))
+  ((DistrictHeating)(DistrictHeatingWater))
```

Comment:
Aliases on FuelType Enum. SHould always use valueDescription() to interface with E+ now